comictalker/talker_utils.py

```diff
@@ -45,6 +45,16 @@ def map_comic_issue_to_metadata(
     metadata.series = utils.xlate(issue_results.series.name)
     metadata.issue = IssueString(issue_results.issue_number).as_string()
 
+    # Rely on comic talker to validate this number
+    metadata.issue_count = utils.xlate_int(issue_results.series.volume)
+
+    # TODO Generate list in utils (same as language and countries) and check against that in separate PR
```

Comment:
As in my other comment format is a free-form field with some suggested defaults it will probably never be enforced